spec/factories/factory_event.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
```

Comment:
Please, change the filename to be just `event.rb`.
Since it is in `../factories` folder, we know that event is a factory.